### PR TITLE
Fix and improvement on Loan module

### PR DESCRIPTION
**Pull request by indelog**
* Fix page redirection on loan delete.
* Add dedicated status for loan with started payment (show if payment is
  started on loan card).
* Add fk_payment_loan property for loan_schedule and add link to loan payment
  on shedule page for paid line.
* Make it not possible to use schedule if start loan payment without it (else
  values in schedule can't be good).
* If modify capital in payment with schedule set, automatically updates it
  with the new value.
* Can't modify interest in payment if schedule is used (causes the
  inconsistency for value in schedule).
* User right verification on schedule page.

### CODE REVIEW

File: htdocs/core/lib/loan.lib.php [hunk 1]
```diff
@@ -82,3 +83,50 @@ function loan_prepare_head($object)
...
+ * Calculate remaining loan mensuality and interests
+ *
+ * @param   int     $mens		Value of this mensuality (interests include, set 0 if we don't paid interests for this mensuality)
+ * @param   float   $capital    Remaining capital for this mensuality
+ * @param   float   $rate		Loan rate
+ * @param   int     $echance	Actual loan term
+ * @param   int   	$nbterm  	Total number of term for this loan
+ * @return  array				Array with remaining capital, interest, and mensuality for each remaining terms
+ */
+function loan_calcmens($mens, $capital, $rate, $echance, $nbterm)
```
Comment (eldy) [inline htdocs/core/lib/loan.lib.php:97]: Even if this is not cas for old method name, for all new method introduced, we must follow the camelCase naming convention.
Can you rename into

loanCalcMonthlyPayment

Comment (indelog) [reply htdocs/core/lib/loan.lib.php:97]: I do that today.

Comment (indelog) [reply htdocs/core/lib/loan.lib.php]: Done.

**Comment by indelog** [comment]
@eldy  OK... Big fail !
I did this too quickly .
Il think i will do a new PR.
Is it possible to delete this one ?

**Pull request closed without merging**